linux_os/guide/system/permissions/partitions/mount_option_dev_shm_nodev.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora,ol7
```

Comment:
Ubuntu:

```tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,size=503544k,nr_inodes=125886)```

SUSE-like: See post 3: https://forums.opensuse.org/showthread.php/526568-running-out-of-disk-space-on-root-directory